examples/uv_actor.lua

```diff
@@ -0,0 +1,48 @@
+-- Use LibUV event loop in actor thread
+--
+-- !NOTE! I can not figure out why this does not work
+--  with `inproc` transport so I use `tcp` to communicate
+--  with actor thread
```

Comment:
Checkout this example https://gist.github.com/moteus/fc8324c12770f266666b
Try remove `cli:recv(zmq.DONTWAIT)` or change `ep` to tcp